session.go

```diff
@@ -83,6 +83,12 @@ type Session interface {
 	// the request handling loop. Registering nil will unregister the channel.
 	// During the time that no channel is registered, breaks are ignored.
 	Break(c chan<- bool)
+
+	// DisablePTYEmulation disables the default PTY emulation provided by the session
+	// when no external PTY management is present.
```

Comment:
Maybe:

// DisablePTYEmulation disables the session's default minimal PTY emulation. If you're setting the pty's termios settings from the Pty request, use this method to avoid corruption.

*shrug*